source/likely.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
 var likely = require('./index.js');
+var history = require('./history');
 
-window.addEventListener('load', likely.initiate);
+window.addEventListener('load', function () {
+    likely.initiate();
+    history.init();
```

Comment:
There’s a minor problem here with different naming of the initialization methods, but I don’t know how to fix it at the moment. I don’t like `initiate` so I wouldn’t rename `history.init()` to it, and we can’t change the public API to rename `likely.initiate()` to `init` (we _can_ introduce a new method and deprecate the older one, but it’s another story). Let’s leave this for now as-is.